doc/lsst.pipe.base/command-line-task-data-repo-howto.rst

```diff
@@ -0,0 +1,136 @@
+.. _command-line-task-data-repo-howto:
+
+#################################################################
+Using Butler data repositories and reruns with command-line tasks
+#################################################################
+
+:ref:`Command-line tasks <using-command-line-tasks>` use Butler data repositories for reading and writing data.
+Butler data repositories are specially-configured directories on a filesystem.
```

Comment:
@ktlim are there specific tickets for creating repository manipulation utilities? I recently asked on `#dm` how to copy a repository, and got a very directory-specific answer; I'd like to upgrade to a more generic approach once it exists.